deploy/olm-catalog/eunomia/0.1.0/eunomia.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,286 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
So this file is that got merged into repo https://github.com/operator-framework/community-operators. Do we actually need to keep this in our repo?